.ruby-version

```diff
@@ -1 +1 @@
-2.1.3
+2.3.1
```

Comment:
Ah yes. It's not hard to update the virtual machine, but I would wait until that update has happened before updating the ruby version in this file. It should not be the case that other developers need to keep switching back and forth, though; this file really doesn't change, so it can be ignored for git purposes.

We can talk after I get back next week about updating the virtual machine.